projecty/src/main/java/com/resource/TweetController.java

```diff
@@ -1,56 +1,60 @@
 package com.resource;
 
-import Model.TwitterData;
-import com.service.Service;
+import com.Model.SendResponse;
+import com.Model.TwitterData;
+import com.service.Services;
 import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 import twitter4j.Status;
-import twitter4j.TwitterException;
 import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
 import java.util.List;
 
 @Produces(MediaType.APPLICATION_JSON)
-@Path("/api/1.0/twitter")
+@RestController
 public class TweetController {
     Logger logger = LoggerFactory.getLogger(TweetController.class);;
-    Service service ;
 
-    public TweetController(){
-        service = new Service();
-    }
+    @Autowired
+   Services services;
 
-    @POST
-    @Path("/sendTweet")
-    public Response sendTweet(PostReq postReq) throws TwitterException {
-        String tweet = postReq.getMessage();
-        if (StringUtils.isEmpty(tweet)) {
-            logger.error("Field is empty");
-            return Response.status(400, "Field is empty").build();
-        } else {
-            Status status = service.sendTweets(tweet);
-            if (status.getText().equals(tweet)) {
-                logger.info("Tweet posted Successfully");
-                return Response.status(200, "Tweet posted Successfully").build();
+    @PostMapping("/sendTweet")
+    public SendResponse sendTweet(@RequestBody PostReq postReq)  {
+        try {
+            String tweet = postReq.getMessage();
+            if (StringUtils.isEmpty(tweet)) {
+                logger.error("Field is empty");
+                return new SendResponse("Field is empty",400);
             } else {
-                logger.error("Request incomplete");
-                return Response.status(500, "Request incomplete").build();
+                Status status = services.sendTweets(tweet);
+                if (status.getText().equals(tweet)) {
+                    logger.info("Tweet posted Successfully");
+                    return new SendResponse("Tweet posted Successfully",200);
+                } else {
+                    logger.error("Request incomplete");
+                    return new SendResponse("Request incomplete",500);
```

Comment:
https://khoros.slack.com/archives/C02JGBKPMKK/p1638184739007200

please apply this

Reply:
I have Applied